static/js/public/first-snap-flow.js

```diff
@@ -183,7 +183,87 @@ function push() {
   });
 }
 
+function updateNotification(notificationEl, className, message) {
+  notificationEl.className = className;
+  notificationEl.querySelector(".p-notification__response").innerHTML = message;
+}
+
+function successNotification(notificationEl, message) {
+  updateNotification(notificationEl, "p-notification--positive", message);
+}
+
+function errorNotification(notificationEl, message) {
+  updateNotification(notificationEl, "p-notification--negative", message);
+}
+
+function validateSnapName(name) {
```

Comment:
I know we're not testing the public stuff so much, but would be good to split this out into a module (potentially for other "snap validation" stuff in the future) and add some tests around it.

Reply:
Yes, I already want to reuse it in 'choose snap name' UI I have in progress, so will extract it and test it there.